src/const.h

```diff
@@ -542,7 +540,6 @@ enum PlayerFlags : uint64_t {
 	PlayerFlag_SetMaxSpeed = 1 << 29,
 	PlayerFlag_SpecialVIP = 1 << 30,
 	PlayerFlag_NotGenerateLoot = static_cast<uint64_t>(1) << 31,
-	PlayerFlag_CanTalkRedChannelAnonymous = static_cast<uint64_t>(1) << 32,
```

Comment:
Good place to leave a comment here that such a thing existed at `<< 32` and was removed.

Reply:
I agree. I've sent a new commit adding a comment about it on a new line right after `PlayerFlag_NotGenerateLoot = static_cast<uint64_t>(1) << 31,`.

Reply:
I think the proper word would be "exponent 32", not "exponential" (adjective)

Reply:
Oops, my bad. Fixed.